README.md

```diff
@@ -0,0 +1,52 @@
+Yocto Layer for Crosswalk
+============================
+
+This layer adds the packages necessary for adding support for [Crosswalk Project](https://crosswalk-project.org/) to your Yocto distribution.
+
+## Supported Yocto Configurations
+* [Ostro™ OS XT](https://github.com/ostroproject/ostro-os-xt)
+
+Usages
+=====
+### Ostro™ OS XT
+#### Build Ostro™ OS XT image
+Please refer to [Building Ostro™ OS XT](https://github.com/ostroproject/ostro-os-xt#building-ostro-tm-os-xt) to setup and build the base Ostro™ OS XT image, assuming the `ostro-os-xt` is checked out at `/path/to/ostro-os-xt` folder.
+
+#### Build Ostro™ OS XT image with Crosswalk layer
+Checkout the `meta-crosswalk` to local folder, assuming `/path/to/meta-crosswalk`.
+
+Add the `meta-crosswalk` layer to `/path/to/ostro-os-xt/build/conf/bblayers.conf`
+```bitbake
+OSTRO_LAYERS += "/path/to/meta-crosswalk"
+```
```

Comment:
An alternative is the `bitbake-layers add-layer <path/to/meta-crosswalk>` command.